### PR TITLE
Sara/SBOM and lang updates nov

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [x] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

### CODE REVIEW

**Review by milanwilliams — APPROVED** [review]

File: docs/semgrep-supply-chain/sbom.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## Generating and downloading an SBOM for a single project
+
+:::info Prerequisites
+- **SBOM generation** can be performed only through Semgrep Cloud Platform (SCP). [Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to use this feature.
+- You need at least 1 successful Supply Chain scan on the **trunk branch** of each repository you want to generate an SBOM for. To add a repository to Semgrep for scanning, see [<i class="fa-regular fa-file-lines"></i> Running scans](/semgrep-cloud-platform/getting-started/#running-scans).
+:::
+
+1. In Semgrep Cloud Platform, click **Supply Chain > Dependencies**. 
+2. Click the **Download <i class="fa-solid fa-download"></i>** icon next to the repository you want an SBOM for.
+3. Click the format you want the SBOM to be in. After clicking, Do not refresh or leave the page.
```
Comment (neo413001) [inline docs/semgrep-supply-chain/sbom.md:31]: "do not" instead of "Do not"

File: docs/semgrep-supply-chain/sbom.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- CycloneDX 1.4 XML
+
+## Generating and downloading an SBOM for a single project
+
+:::info Prerequisites
+- **SBOM generation** can be performed only through Semgrep Cloud Platform (SCP). [Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to use this feature.
+- You need at least 1 successful Supply Chain scan on the **trunk branch** of each repository you want to generate an SBOM for. To add a repository to Semgrep for scanning, see [<i class="fa-regular fa-file-lines"></i> Running scans](/semgrep-cloud-platform/getting-started/#running-scans).
+:::
+
+1. In Semgrep Cloud Platform, click **Supply Chain > Dependencies**.
```
Comment (neo413001) [inline docs/semgrep-supply-chain/sbom.md:29]: going to dependencies tab means you can download SBOM per project AFAIK. 
If the user wants to download SBOM for *all* the projects, then should we ask them to go the to the "Vulnerabilities" tab and hit the SBOM button?
cc: @andy-r2c

**Review by neo413001 — CHANGES_REQUESTED** [review]

File: docs/semgrep-supply-chain/sbom.md (hunk 2 above)
Comment (s-santillan) [reply docs/semgrep-supply-chain/sbom.md:29]: the "All projects" feature doesn't work yet :)

File: docs/semgrep-supply-chain/sbom.md (hunk 1 above)
Comment (s-santillan) [reply docs/semgrep-supply-chain/sbom.md]: Done

**Review by neo413001 — APPROVED** [review]

**Pull request merged**